tadasets/swiss_holes_helpers.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+
+def in_a_hole(row, centers, radiuses):
+    """Helper function.
+
+    Parameters
+    ============
+
+    row: entry in matrix
+        One datapoint row.
+    centers: np.ndarray with shape (n_holes, d)
+        All the center coordinates of a ball.
+    radiuses: np.ndarray with shape (n_holes, )
+        All the radiuses of all the holes.
+    """
+    return any(np.apply_along_axis(np.linalg.norm,1,row - centers) <= radiuses) is False
```

Comment:
not sure if it's the case, but would `np.any` be faster than the native python `any`?

Reply:
They also behave differently.
```python
import numpy as np
data = [[False]]
np.any(data)  # False
any(data)  # True
```
I recommend `return not np.any(...)` instead of `return any(...) is False`.